test/unit-test/ConfigFiles/FreeRTOSIPConfig.h

```diff
@@ -36,8 +36,6 @@
 
 #define ipconfigUSE_IPV4            ( 1 )
 
-#define ipconfigUSE_IPV6            ( 1 )
-
```

Comment:
Do we not need ipconfigUSE_IPV6 flag here

Reply:
we are not globally enabling ipv6 flag, instead only for tests that require ipv6 flags to make it build